src/transformers/__init__.py

```diff
@@ -1361,46 +1379,192 @@
     from .models.distilbert import DISTILBERT_PRETRAINED_CONFIG_ARCHIVE_MAP, DistilBertConfig, DistilBertTokenizer
     from .models.dpr import (
         DPR_PRETRAINED_CONFIG_ARCHIVE_MAP,
+        TF_DPR_CONTEXT_ENCODER_PRETRAINED_MODEL_ARCHIVE_LIST,
+        TF_DPR_QUESTION_ENCODER_PRETRAINED_MODEL_ARCHIVE_LIST,
+        TF_DPR_READER_PRETRAINED_MODEL_ARCHIVE_LIST,
         DPRConfig,
         DPRContextEncoderTokenizer,
```

Comment:
Do you know why all of these are added to the initialization? They're already imported below:
https://github.com/huggingface/transformers/blob/a01ea31b5c605ea812777330ed1de8e805dca9ea/src/transformers/__init__.py#L2096-L2098

Reply:
(this applies to all of the `__init__.py` except the changes related to layoutlm to the init)

Reply:
I am not sure why they are being added here, seems like the last commit `Style` added them there

Reply:
I guess I have to manually remove them, right?

Reply:
Yes or we can take care of that for you that's not an issue, it's weird that `make style` added everything here.

Reply:
I think I addressed all the reviews comments except this wired imports additions to initialization file , plus it has conflict. I was wondering if you could fix this. Given my two field attempts to rebase in the past I thought it'd be quicker and less painful for you :-)